types/twine-sugarcube/macro.d.ts

```diff
@@ -69,7 +70,13 @@ export interface MacroContext {
      * The (execution) context object of the macro's parent, or null if the macro has no parent.
      * @since 2.0.0
      */
-    parent: object;
+    parent?: object;
```

Comment:
The `parent` property is not optional—as I understand TS optional.  It will always be either a context object or `null`, never missing or `undefined`.

This should be reverted.

Reply:
Correct. I misread it.